frontend/src/pages/ActivityReport/Pages/nextSteps.js

```diff
@@ -158,7 +158,14 @@ const NoteEntries = ({ name, humanName, label }) => {
           : (
             <Button type="button" unstyled onClick={() => onEdit(notes.length)}>
               <FontAwesomeIcon icon={faPlusCircle} />
-              <span className="padding-left-05">Add New Next Step</span>
+              <span className="padding-left-05">
+                Add New Next
```

Comment:
This reads as `Add new next specialist step`, probably want `Add new specialist next step`